packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -307,26 +400,40 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
         <div
           ref={selectRef}
           className={cx('iui-select-button', {
-            'iui-placeholder': !selectedItem && !!placeholder,
+            'iui-placeholder':
+              (!selectedItems || selectedItems.length === 0) && !!placeholder,
             'iui-disabled': disabled,
             [`iui-${size}`]: !!size,
           })}
           onClick={() => !disabled && toggle()}
           onKeyDown={(e) => !disabled && onKeyDown(e, toggle)}
           tabIndex={!disabled ? 0 : undefined}
         >
-          {!selectedItem && <span className='iui-content'>{placeholder}</span>}
-          {selectedItem &&
-            selectedItemRenderer &&
-            selectedItemRenderer(selectedItem)}
-          {selectedItem && !selectedItemRenderer && (
-            <>
-              {selectedItem?.icon &&
-                React.cloneElement(selectedItem.icon, {
-                  className: cx(selectedItem?.icon.props.className, 'iui-icon'),
-                })}
-              <span className='iui-content'>{selectedItem.label}</span>
-            </>
+          {(!selectedItems || selectedItems.length === 0) && (
+            <span className='iui-content'>{placeholder}</span>
+          )}
+          {isMultipleEnabled(selectedItems, multiple) ? (
+            <MultipleSelectButton
+              selectedItems={selectedItems}
+              selectedItemsRenderer={
+                selectedItemRenderer as (
+                  options: SelectOption<T>[],
+                ) => JSX.Element
+              }
+              onChange={(value, event, isKeyboard) => {
+                if (isKeyboard) {
+                  focusPreviousElement(selectRef.current);
```

Comment:
It is not focusing previous tag on removal.

Reply:
should be focusing firstly next and then previous, if not available